frontend/src/views/GameInvite.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+	<div class="page_box_wrapper">
+		<div class="page_box">
+			<h1>YOU'VE BEEN INVITED</h1>
+			<h1>BY</h1>
+			<h1>PLAYER_02</h1>
+			<div class="c_block c_form_group">
+				<div class="c_field_group">
+					<InputField label="score_to_win" id="disabled_field" v-model="score_to_win"/>
+					<InputField label="background_color" value="disabled field" v-model="background_color"/>
```

Comment:
the id is "disabled_field" but the field is not disabled. Does that have a reason? 
It also seems like the fields should actually be disabled.

Reply:
this is only a quick the setup, I'll be working of game-invite in the next PR